asab/metrics/service.py

```diff
@@ -16,27 +16,32 @@
 
 #
 
+
 def metric_dimension(metric_name, tags):
 	dim = metric_name
 	if tags is not None:
 		for k in sorted(tags.keys()):
 			dim += ',{}={}'.format(k, tags[k])
 	return dim
 
-#
 
-class MetricsService(asab.Service):
+class MetricsService(asab.Service, asab.ConfigObject):
```

Comment:
I guess, Service is not a ConfigObject - you should go with `asab.Config.add_default`